public/locales/en/modules/smart-home/entity-state.json

```diff
@@ -0,0 +1,17 @@
+{
+    "entityNotFound": "Entity not found",
+    "descriptor": {
+        "name": "Smart home entity",
```

Comment:
Widget description is way too vague, this works with all smart home entities, isn’t it just home assistant ? Can I connect my google assistant smart lightbulb ?